app/assets/javascripts/components/revisions/diff_viewer.jsx

```diff
@@ -156,11 +161,27 @@ const DiffViewer = React.createClass({
     const wikiDiffUrl = this.webDiffUrl();
 
     let diffComment;
-    // Only show the edit summary for a single revision; for cumulative diffs,
-    // the edit summary is for the *parent* of the first revision, so it's
-    // irrelevant and confusing to show that.
+    let revisionDateTime;
+    let firstRevTime;
+    let lastRevTime;
+    let timeSpan;
+
+    // Edit summary for a single revision:
+    //  > Edit date and number of characters added
+    // Edit summary for range of revisions:
+    //  > First and last times for edits to article (from first applicable rev to last)
     if (!this.props.first_revision) {
-      diffComment = <p className="diff-comment">{this.state.comment}</p>;
+      revisionDateTime = moment(this.props.revision.date).format('YYYY/MM/DD h:mm a');
+      diffComment = <p className="diff-comment">{this.state.comment}&nbsp;
```

Comment:
This is looking pretty clean now! I'll give it a spin as soon as I can. After this change, `diffComment` is doing a more than the name implies. It was named that because it just showed the comment (aka, the edit summary) left along with the diff.